doc/start.md

```diff
@@ -2,20 +2,27 @@
 @parent can-route.static
 @release 3.3
 
-Initializes can-route.
+@description Initializes can-route.
 
 @signature `route.start()`
 
-Sets up the two-way binding between the hash and the can-route observable
-map and sets the route map to its initial values.
+  Sets up the two-way binding between the hash and the can-route observable map and sets the route map to its initial values.
```

Comment:
We should include information about how this works.  Who wins in a "fight" between the URL data and the `route.data`?

```
route.data.assign({foo: "bar"})
window.location.href = "#!zed=ted";
route.start() //-> What happens? (I think the url wins, but data might also be merged)
```

Reply:
url data does win, it's not merged.